petstore/src/main/scala/io/finch/petstore/reader.scala

```diff
@@ -0,0 +1,25 @@
+package io.finch.petstore
+
+import io.finch.request._
+
+/**
+ * Represents a reader object that helps extract parameters from query params and bodies.
+ */
+object reader {
+  /**
+   *  Reads JSON representing a Status from a given query parameter.
+   */
+  implicit val statusReader: RequestReader[Status] = param("status").map(Status.fromString)
+
+  /**
+   *  Reads JSON representing a Tag from a given query parameter.
+   */
+  implicit val tagReader: RequestReader[Seq[String]] = param("tags").map { tags =>
```

Comment:
This was my mistake originally, but I think @vkostyukov mentioned that `params("status")` should work here for comma-separated values.